controllers/bootstrap/init.go

```diff
@@ -1420,3 +1420,56 @@ func CheckClusterType(mgr manager.Manager, ns string) (bool, error) {
 		return true, nil
 	}
 }
+
+func (b *Bootstrap) DeployCertManagerCR() error {
+	deployedNs := b.CSData.MasterNs
+	if b.MultiInstancesEnable {
+		deployedNs = b.CSData.ControlNs
+	}
+	_, err := b.GetSubscription(ctx, constant.CertManagerSub, deployedNs)
+	if errors.IsNotFound(err) {
+		klog.Infof("Skipped deploying cert manager CRs, %s not installed yet.", constant.CertManagerSub)
```

Comment:
I don't wanna common service CR be reconciled over and over again if cert-manager operator is not deployed yet.

So we watch the creation and update of cert-manager operator subscription to trigger the reconciliation of commonservice CR. And once subscription is there, it will wait until the APIs are available.

Reply:
I see. It makes sense